### ISSUE TITLE
Inconsistent `{storm}`: <Incorrect filepath in DOC.md for storm>

### ISSUE THREAD

**Issue by adityakiran1423** [DiceDB/dice#720]
## Steps to reproduce

Running the command `go run storm/set/main.go` will result in an error,. No file called main.go in storm/set folder (line 19 DOC.md)

current path in DOC.md to run storm : `go run storm/set/main.go`
revised path required to run storm : `go run storm/set/set.go`

## Expected output

The expected path to run storm should be : 

```
go run storm/set/set.go
```

## Expectations for resolution

This issue will be considered resolved when the following things are done

1. path is changed from `storm/set/main.go` to `storm/set/set.go`

You can find the tests under the `tests` directory of the [`dice`](https://github.com/dicedb/dice) repository and the steps to run are in the [README file](https://github.com/dicedb/dice). Refer to the following links to set up DiceDB and Redis 7.2.5 locally

- [setup DiceDB locally](https://github.com/dicedb/dice)
- [setup Redis 7.2.5 locally](https://gist.github.com/arpitbbhayani/94aedf279349303ed7394197976b6843)
- [setup DiceDB CLI](https://github.com/dicedb/dice)

**Comment by adityakiran1423** [comment]
Is it possible to assign the issue to myself?

**Comment by shardul08** [comment]
> created branch issue-720, need authorization to push code. Getting an error if done otherwise.

@adityakiran1423 you need to fork the repo and make changes in your copy and raise the PR to this repo. Please follow  [CONTRIBUTING.md](https://github.com/DiceDB/dice/blob/master/CONTRIBUTING.md) for detailed instructions.

**Comment by adityakiran1423** [comment]
@shardul08 , apologies for the  context, I meant to say that I am not able to push to my branch (issue-720). I looked up [CONTRIBUTING.md](https://github.com/DiceDB/dice/blob/master/CONTRIBUTING.md), I was going to raise a PR after I pushed to my branch, however I ran into this : 

![issue720](https://github.com/user-attachments/assets/388120ed-369f-46b7-8622-31eb631fbe89)

I was wondering if you could help me with this 
Thanks
-Aditya

**Comment by shardul08** [comment]
Seems like you are trying to push the branch to DiceDB repo. You will need to create a fork of DiceDB repo, create a new branch in the forked repo, make changes and push it. Once the new branch is pushed to the forked repo you will be able to raise the PR.

**Comment by adityakiran1423** [comment]
Yep, totally forgot to fork the repo, apologies for the oversight 😬
Followed the instructions are raised a PR
[Link to the PR](https://github.com/DiceDB/dice/pull/721)
Thanks for the help
-Aditya

**Comment by adityakiran1423** [comment]
@shardul08, Hello, was wondering if there is any update on [PR : #721](https://github.com/DiceDB/dice/pull/721)
